gusto/core/io.py

```diff
@@ -531,7 +531,11 @@ def setup_dump(self, state_fields, t, pick_up=False):
 
         # dump initial fields
         if not pick_up:
-            self.dump(state_fields, t, step=1)
+            step = 1
+            last_ref_update_time = None
+            initial_steps = None
+            time_data = (t, step, initial_steps, last_ref_update_time)
```

Comment:
I feel slightly concerned about having a tuple of things that I might need to remember the order of! Could we use a [named tuple](https://docs.python.org/3/library/collections.html#collections.namedtuple)?

Something like (completely untested!):
```
TimeData = namedtuple('TimeData', ['t', 'step', 'initial_steps', 'last_ref_update_time'])
time_data = TimeData(t, step, initial_steps, last_ref_update_time)
```
Then you can later access these like: `t = time_data.t`.